adoc/desserts/chocolate_muffins_with_liquid_core.adoc

```diff
@@ -0,0 +1,43 @@
+[id='sec.chocolate_muffins_with_liquid_core']
+
+ifdef::env-github[]
+:imagesdir: ../images
+endif::[]
+ifndef::env-github[]
+:imagesdir: images
+endif::[]
+
+
+=== Chocolate Muffins with liquid core
+image::chocolate_muffins_with_liquid_core/IMG_0853.jpg[Chocolate Muffin, 640, 480]
+
+Contributed by: Robert 
+image:contributors/robert_s.png[Robert]
```

Comment:
you may want to put size constraints on this, it fills the whole page in the preview!